frontend/public/src/lib/util_test.js

```diff
@@ -216,7 +216,6 @@ describe("utility functions", () => {
   })
 
   it("formatPrettyDate should return a formatted moment date", () => {
-    moment.locale("en")
```

Comment:
`en` is the default locale. There are now no calls to `locale()` in the codebase.